.gitignore

```diff
@@ -44,7 +44,6 @@ python/nvfuser_direct/*.so
 *.swp
 *~
 .~lock.*
-.vscode
```

Comment:
I don't personally have any `.vscode` on my environment, but someone might have their configs. Removing this from `.gitignore` could make `git add .` accidentally commit people's config.